mteb/tasks/PairClassification/mlt/PawsXMaltese.py

```diff
@@ -0,0 +1,46 @@
+from __future__ import annotations
+
+from mteb.abstasks.AbsTaskPairClassification import AbsTaskPairClassification
+from mteb.abstasks.TaskMetadata import TaskMetadata
+
+
+class PawsXMaltese(AbsTaskPairClassification):
+    metadata = TaskMetadata(
+        name="PawsXMaltese",
```

Comment:
I agree that this makes more sense indeed. Can a task refer to multiple paths in hugging face so that I can refer to "PAWS-X-maltese" in PawsX and do the merge at the load data step? Or do I have to create a new dataset from "paws-x" and "PAWS-X-maltese", upload it to HF, to replace the one currently in PawsX?

Reply:
@wissam-KH you could create a copy of the original PAWSX and add a Maltese subset in mteb on hugging face, you can add a citation in the dataset card. Just as we did with these 2 datasets: https://huggingface.co/datasets/mteb/multilingual-scala-classification, https://huggingface.co/datasets/mteb/multilingual-sentiment-classification

Reply:
@wissam-KH  if this dataset is machine-translated, it may be better to not mix it with the original PAWS-X, see the discussion in this [PR](https://github.com/embeddings-benchmark/mteb/pull/649). It would suggest to have a new task named `PawsXTranslated`. Let me know what you think?

Reply:
At the moment I don't believe we accept machine-translated (unless verified) dataset: https://github.com/embeddings-benchmark/mteb/blob/main/docs/adding_a_dataset.md#2-creating-the-metadata-object

Reply:
There's not much information about the dataset, so maybe it's better not to risk it. I'd rather close the PR then, thanks for reviewing. 👍 
Note: I picked it up from here quickly https://github.com/embeddings-benchmark/mteb/issues/419

Reply:
Thanks @wissam-sib - I have removed it from #419